src/test/java/edu/uci/ics/cs221/index/inverted/Team1StressTest.java

```diff
@@ -0,0 +1,146 @@
+package edu.uci.ics.cs221.index.inverted;
+
+import edu.uci.ics.cs221.analysis.Analyzer;
+import edu.uci.ics.cs221.analysis.NaiveAnalyzer;
+import edu.uci.ics.cs221.storage.Document;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.*;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.*;
+
+/**
+ * Stress test cases for inverted index by team 1.
+ *
+ * @author Zixu Wang
+ */
+public class Team1StressTest {
+    private InvertedIndexManager iim;
+    private Analyzer analyzer = new NaiveAnalyzer();
+    private String indexDir = "index/Team1StressTest";
+
+    private Document[] largeDocs = new Document[] {
+            generateDoc(5),
+            generateDoc(5)
+    };
+
+    private Document[] manyDocs = new Document[] {
+            generateDoc(1),
+            generateDoc(1),
+            generateDoc(1),
+            generateDoc(1),
+            generateDoc(1),
+            generateDoc(1),
+            generateDoc(1),
+            generateDoc(1),
+            generateDoc(1),
+            generateDoc(1)
+    };
+
+    @Before
+    public void initialize() {
+        iim = InvertedIndexManager.createOrOpen(indexDir, analyzer);
+        iim.DEFAULT_FLUSH_THRESHOLD = 1;
+        PageFileChannel.resetCounters();
+    }
+
+    @After
+    public void clean() throws IOException {
+        deleteDirectory(Paths.get(indexDir));
```

Comment:
```suggestion
        deleteDirectory(Paths.get(indexDir));
        InvertedIndexManager.DEFAULT_FLUSH_THRESHOLD = 1000;
```
I guess you should change back the `DEFAULT_FLUSH_THRESHOLD` since this is a static variable. Other Tests might use this again.